packages/fiber/src/web/index.tsx

```diff
@@ -178,7 +178,7 @@ function createPortal(
 
 reconciler.injectIntoDevTools({
   bundleType: process.env.NODE_ENV === 'production' ? 0 : 1,
-  rendererPackageName: 'react-three-fiber',
+  rendererPackageName: '@react-three/fiber',
```

Comment:
@drcmda can you change the name in `test-renderer` if It hasn't been done already?